spec.html

```diff
@@ -174,12 +174,12 @@ <h1>Normative References</h1>
   <p>The following referenced documents are indispensable for the application of this document. For dated references, only the edition cited applies. For undated references, the latest edition of the referenced document (including any amendments) applies.</p>
   <p>ISO/IEC 10646 <i>Information Technology — Universal Multiple-Octet Coded Character Set (UCS) plus Amendment 1:2005, Amendment 2:2006, Amendment 3:2008, and Amendment 4:2008</i>, plus additional amendments and corrigenda, or successor</p>
   <p>
-    ECMA-402, <i>ECMAScript 2015 Internationalization API Specification</i>.<br>
-    <a href="https://ecma-international.org/publications/standards/Ecma-402.htm">https://ecma-international.org/publications/standards/Ecma-402.htm</a>
+    ECMA-402, <i>ECMAScript Internationalization API Specification</i>.<br>
+    <a href="https://www.ecma-international.org/publications-and-standards/standards/ecma-402/">https://www.ecma-international.org/publications-and-standards/standards/ecma-402/</a>
```

Comment:
should we maybe link to tc39.es instead of the yearly snapshot?

Reply:
This was marked resolved, but without an explanation or an update?

Reply:
We should like to the Ecma page that lists the yearly releases so that the appropriate one can be chosen.

> specifically the annual edition corresponding to this edition of this specification

Reply:
They're both living standards - the appropriate one is only, and always, the one on tc39.es, imo.

Reply:
I think we had this discussion already...
1) There is only one TC39 Standard (ES20xx) that is currently approved by the Ecma GA year by year in their June meeting.  This is what is reflected on the Ecma website, where you can access and download those standards. TC39 and the ECMA GA wish was (since 2016) to do this once per year (By the way, In theory one could go to two approvals per year, per GA, every 6 months. Theoretically, one could place letter ballot approvals in between it was needed. Theory, Ecma never did that). According to Ecma terms the actual Edition of ES20xx is the only valid standard.
2) There are between the two Editions of ES20xx so called "Draft Standards". This just reflects work in progress, but they are not the formally approved Ecma standards, because formal approval is on the GA level by Ecma Ordinary Members. Draft standards - which are not voted by the GA - can be changed through the TC work any time both from the substance and the editorial point of view. This is what TC39 does. To call such "draft standards" as "living standards" to make the impression (and even declare it so) that the draft is already the formally approved standard is not a concept what the Ecma procedures support. Let alone two "living standards" in parallel, which are different.

Reply:
@ljharb 

@bakkot and I already discussed this and approved the current wording. Please merge as-is.